core/files/files.go

```diff
@@ -471,7 +389,20 @@ func (s *service) getContentReader(ctx context.Context, spaceID string, file *st
 	return dec.DecryptReader(fd)
 }
 
-var errFileExists = errors.New("file exists")
+type errFileExists struct {
+	fileId domain.FileId
```

Comment:
May be not store fileId in error and return fileId separately? Seems like it's not relevant field to error

Reply:
Thank you! Introduced result structure to avoid previous weird error handling